Automattic/simplenote-ios
# Nü Note Options: Mark V

### 🛑  Important
Don't merge until #906 #907 #909 and #910 are ready: everything must be merged in a single shot!

### Details:
- **Wires Collaboration support** in the Note Options Interface
- **OptionsViewController** now implements the AddCollaborator delegate protocol
- **Editor:** Drops the **blankNote** flag
- **Note:** New **isBlank** and **emailTags** properties

@frosty Sir!! Can I bug you with this one?
Thanks in advance!!!

Ref. #763

### Test: Collaboration
1. Open any note
2. Press over the top `( i )` icon
3. Press over **Collaborate**

- [ ] Verify you can add / remove collaborators
- [ ] Verify that newly added collaborators get access to the shared document

### Test: Empty Notes
1. Launch Simplenote and press the top right `+` button
2. **Do not type anything**
3. Press back

- [ ] Verify that the newly added note is deleted

### Test: Non Empty Notes
1. Launch Simplenote and press the top right `+` button
2. Type anything
3. Delete the contents
4. Press back

- [ ] Verify that the newly added is **not** deleted


### Release
`RELEASE-NOTES.txt` was updated in cdedacd with:
 
> New Note Options interface #763
